### PR TITLE
final(?) issue batch

### LINKED ISSUE #2176: Contact Form Submission - Suggestion
Someone submitted the contact form!

**Module**: Gold - String Hashing
**Topic**: Suggestion
**Message**: 
Add this old silver hashing problem:http://www.usaco.org/index.php?page=viewproblem2&cpid=529

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

should close #2176 and #2583

### CODE REVIEW

File: solutions/cses-1163.mdx [hunk 1]
```diff
@@ -30,44 +30,44 @@ multiset.
...
+	std::ios::sync_with_stdio(false);
+	std::cin.tie(NULL);
+
+	int street_len;
+	int light_num;
+	cin >> street_len >> light_num;
+
+	set<int> lights{0, street_len};
+	multiset<int> dist{street_len};
+	for (int l = 0;l < light_num; l++) {
```
Comment (ryanchou-dev) [inline solutions/cses-1163.mdx:45]:
```suggestion
	for (int l = 0; l < light_num; l++) {
```
omg my first time seeing kevin mess up spacing

**Review by ryanchou-dev — APPROVED** [review]
yes.

File: solutions/cses-1163.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply solutions/cses-1163.mdx:45]: you saw nothing

**Review by TruaShamu — APPROVED** [review]
looks good to me :)

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**